ui/scss/mixin/link.scss

```diff
@@ -9,8 +9,8 @@
     }
   }
 
-  &:not(.no-underline) {
-    text-decoration: underline;
+  &:not(.underline) {
+    text-decoration: none;
```

Comment:
I think this change added underlining to text button icons (e.g. the ones in `FileActions`). I think this can look poor with some icons, so should just be avoided.

Reply:
ok, I'll revert this, but at least header can't have underline text :upside_down_face: